templates/donor-mandate-confirmation.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends renderHtml ? '_base.html.twig' : '_base.text.twig' %}
+
+{% block content %}
+    <h1>Regular Giving Mandate for {{ charityName }}</h1>
+
+    <p>
+    Thank you for supporting {{ charityName }} with your generous gift through Big Give.
+    </p>
+
+    <p>
+    A message from {{ charityName }}:
+    </p>
+    <blockquote>{{ campaignThankYouMessage | nl2br }}</blockquote>
+
+    <p>
+        {{ campaignThankYouEmail }}
+    </p>
+
+    <h2>Regular Giving Mandate Details</h2>
+    <p>
+        Thank you for setting up your regular gift to {{ charityName }} for the {{ campaignName }} campaign. {{ charityName }}
+        is registered with the Charity Commission for England and Wales with charity number {{ charityNumber }}. This email
+        certifies that no goods or services will be provided in return for your contribution.
+    </p>
+
+    <ul>
+        <li>Regular gift sign up date: {{ signupDate}}</li>
+        <li>Donor: {{ donorName }}</li>
+        <li>Schedule: {{ schedule }}</li>
+        <li>Next expected payment date: {{ nextPaymentDate }}</li>
+        <li>Amount: {{ amount }}</li>
+        <li>Gift aid value: {{ giftAidValue }}</li>
+        <li>Total with gift aid: {{totalIncGiftAid }}</li>
```

Comment:
would people want some sort of reference number for the mandate? We could give them the UUID but that's unwieldy, could add a todo note somewhere if we think they'll want it but we don't have one to give.

We could also give them a link to see the mandate on the site. Currently that would have to be just like https://donate-staging.thebiggivetest.org.uk/my-account/regular-giving but it might be better if it was a page dedicated to this individual mandate, which we could create in future.